environment.yaml

```diff
@@ -0,0 +1,6 @@
+name: saturn-client
+channels:
+  - defaults
+dependencies:
+  - python=3.7
+  - requests
```

Comment:
In my opinion, having a development conda env for a package that only relies on requests is overkill. I'd prefer to remove this file and any references to it

Reply:
I have a different dev env for anything that I work on, so this is a convenience for me.